modules/tensor_mechanics/doc/content/source/utils/RankTwoScalarTools.md

```diff
@@ -0,0 +1,249 @@
+# Rank Two Scalar Tools
+
+
+## Description
+
+This is a utility class that provides a set of functions to compute scalar
+quantities for stress or strain tensors. This class is not directly invoked in
```

Comment:
'for stress or strain tensors' -> 'such as invariants and components in specified directions from rank-2 tensors such as stress or strain.'

Reply:
Also `RankTwoScalarTools` is _not_ a class. It is a namespaced collection of helper functions.